.github/workflows/deploy-tag.yml

```
@@ -0,0 +1,56 @@
+name: Deployment
+
+on:
+  push:
+    tags:
+      - "20.*"
+      - "21.*"
+      - "22.*"
+      - "23.*"
+      - "24.*"
+
+jobs:
+
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - run: npm ci
+    - run: npm run build
+    - name: Update extension’s meta
+      run: |
+        export VER=$(echo "$GITHUB_REF" | sed -e 's/refs\/tags\///')
+        echo https://github.com/$GITHUB_REPOSITORY/tree/$VER > distribution/SOURCE_URL
+        npm run version
+    - uses: actions/upload-artifact@v1
+      with:
+        name: built-extension
+        path: distribution
+
+  deploy-to-chrome:
+    needs: build
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/download-artifact@v1
+      with:
+        name: built-extension
+        path: distribution
+    - run: cd distribution && npx chrome-webstore-upload-cli upload --auto-publish
```

This may be possible

> Basic (download to current working directory):

```suggestion
    - run: npx chrome-webstore-upload-cli upload --auto-publish
```

https://github.com/actions/download-artifact#usage

Also it's best to make the version sticky

```suggestion
    - run: npx chrome-webstore-upload-cli@1 upload --auto-publish
```

---

Looking at https://github.com/DrewML/chrome-webstore-upload-cli/blob/master/wrapper.js#L28-L30, wouldn't the CRX now contain other non-essential files like `package.json` and potentially `node_modules`? Or am I missing something?